NuclearData/materialMenu_mod.f90

```diff
@@ -100,6 +100,7 @@ module materialMenu_mod
     real(defReal),dimension(:),allocatable     :: dens
     type(nuclideInfo),dimension(:),allocatable :: nuclides
     type(dictionary)                           :: extraInfo
+    logical(defBool)                           :: hasTMS = .false.
```

Comment:
I would be inclined to keep TMS out of the material menu if possible. For example, if we wanted to do multipole or if we wanted to do MG data. Maybe there is no other obvious way? I suppose we are in the same difficulty for Sab too. This also comes back to temperature being something that is desirable for any generic material to keep track of and ideally not something that automatically triggers TMS ( MG could benefit from having a temperature despite not using TMS).

Reply:
I agree with changing keywords, but I don't think it's such a problem to go through materialMenu. That's how we read everything about the materials. That info is not used during transport because it is stored inside ceNeutronMaterial.